### PR TITLE
s-tui: init at 0.6.0

### PR DESCRIPTION
**Pull request by infinisil**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -19825,6 +19825,8 @@ in {
     propagatedBuildInputs = [ self.cffi ];
   };
 
+  s-tui = callPackage ../tools/system/s-tui { };
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: this is an application and should not be called via `python-packages.nix`.

**Review by FRidh — CHANGES_REQUESTED** [review]

**Comment by Infinisil** [comment]
Fixed that

File: pkgs/tools/system/s-tui/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, pythonPackages }:
+
+pythonPackages.buildPythonPackage rec {
+  name = "s-tui-${version}";
+  version = "0.6.0";
```
Comment (FRidh) [inline pkgs/tools/system/s-tui/default.nix]: The latest release on PyPI is 0.6.2. Please use `fetchPypi` to fetch it.

**Review by FRidh — CHANGES_REQUESTED** [review]
I noticed this latest release as I was checking whether tests were distributed. They are not.

**Comment by Infinisil** [comment]
Using `fetchPypi` and version 0.6.2 now

**Comment by FRidh** [comment]
ugh..commit name was wrong

**Comment by Infinisil** [comment]
Whoops ¯\\_(ツ)_/¯

**Pull request merged**